bin/pycbc_multi_inspiral

```diff
@@ -602,59 +615,52 @@ with ctx:
                             # Calculate the power and autochi2 values for the coinc
                         # indexes (this uses the snr timeseries before the time
                         # delay, so we need to undo it. Same for normalisation)
+                        # Need all out vals to be the same length.
```

Comment:
At what point is this needed?  I am wondering whether there is a cleaner solution.